firstvoices/backend/models/__init__.py

```diff
@@ -22,6 +22,7 @@
 from .mtd import MTDExportFormat  # noqa F401
 from .page import SitePage  # noqa F401
 from .part_of_speech import PartOfSpeech  # noqa F401
+from .signals.mtd_signals import *  # noqa F401 - MTD Index signals
```

Comment:
Why do these need to be imported here?

Reply:
As discussed, this import of signals has now been moved to the `apps.py` in the `ready()` method of the `BackendConfig`.